content/pillow/concepts/image/terms/transpose/transpose.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: '.transpose()'
+Description: 'Method in the Pillow library used to flip or rotate an image.'
+Subjects:
+  - 'Code foundation'
+  - 'Computer science'
+Tags:
+  - 'Methods'
+  - 'Functions'
+  - 'Libraries'
+  - 'Pillow'
+CatalogContent:
+  - 'learn-Python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+  - 'paths/data-science-foundations'
+---
+
+The **.transpose()** method is used to flip or rotate an image in 90-degree steps.
+
+## Syntax
+
+```pseudo
+Image.transpose(method)
+
+# Where possible values of the method are
+
+PIL.Image.FLIP_LEFT_RIGHT
+PIL.Image.FLIP_TOP_BOTTOM
+PIL.Image.ROTATE_90
+PIL.Image.ROTATE_180
+PIL.Image.ROTATE_270
+PIL.Image.TRANSPOSE or PIL.Image.TRANSVERSE.
```

Comment:
Keep these out of the code block since all of them are not a part of the generic syntax. Also, just below the syntax, we should use a format:

- `Image`: Describe this parameter
- `method`: Describe this parameter and say it has the following types:
  - `FLIP_LEFT_RIGHT`: Flips the image horizontally.
  - and so on
 
So this way you can cover the parameter descriptions.